extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-slider.js

```diff
@@ -156,5 +159,50 @@ describes.realWin(
         .querySelector('.i-amphtml-story-interactive-slider-bubble');
       expect(sliderBubble.textContent).to.be.equal('😄');
     });
+
+    it('should mock a true response and show post-selection state', async () => {
+      env.sandbox
+        .stub(requestService, 'executeRequest')
+        .resolves(getSliderInteractiveData());
+      ampStorySlider.element.setAttribute('endpoint', 'https://example.com');
+      await ampStorySlider.buildCallback();
+      await ampStorySlider.layoutCallback();
+      expect(ampStorySlider.getRootElement()).to.have.class(
+        'i-amphtml-story-interactive-post-selection'
+      );
+    });
+
+    it('should display the selected value in the post-state bubble', async () => {
+      await ampStorySlider.buildCallback();
+      await ampStorySlider.layoutCallback();
+      const slider = ampStorySlider
+        .getRootElement()
+        .querySelector('input[type="range"]');
+      const sliderBubble = ampStorySlider
+        .getRootElement()
+        .querySelector('.i-amphtml-story-interactive-slider-bubble');
+      slider.value = 30;
+      // simulates an input event, which is when the user drags the slider
+      // simulates a change event, which is when the user releases the slider
+      slider.dispatchEvent(new CustomEvent('input'));
+      slider.dispatchEvent(new CustomEvent('change'));
+      expect(sliderBubble.textContent).to.be.equal('30%');
+    });
+
+    it('should display the average indicator in the correct position', async () => {
+      env.sandbox
+        .stub(requestService, 'executeRequest')
+        .resolves(getSliderInteractiveData());
+      ampStorySlider.element.setAttribute('endpoint', 'https://example.com');
+      await ampStorySlider.buildCallback();
+      await ampStorySlider.layoutCallback();
+      const sliderIndicator = ampStorySlider
+        .getRootElement()
+        .querySelector('.i-amphtml-story-interactive-slider-average-indicator');
+      const computedAverageValue = '124.438px';
+      expect(
+        win.getComputedStyle(sliderIndicator).getPropertyValue('left')
```

Comment:
You need to test for the `--average` value to be equal to the average of the votes we load, because it's impossible to understand why left should be equal to `124px`, but the CSS variable is easy to understand.